src/test/java/guitests/UITest.java

```diff
@@ -362,6 +362,29 @@ private void awaitCondition(Callable<Boolean> condition, int timeoutInSeconds) {
     }
 
     /**
+     * Performs logout from File -> Logout on HubTurbo's pView.
+     */
+    public void logout(){
+        clickMenu("File", "Logout");
+    }
+
+    /**
+     * Performs UI login on the dialog box upon startup.
```

Comment:
Does it have to be `upon startup` or it can be anytime the login dialog is displayed?